README.md

```diff
@@ -58,12 +59,14 @@ Before you attempt to make a contribution please read the [Community Participati
 2. Import the project into Android Studio **or** build on the command line:
 
   ```shell
-  ./gradlew clean app:assembleArmDebug
+  ./gradlew clean app:assembleGeckoBetaDebug
   ```
+  
+  Use app:assembleGeckoNightlyDebug to build with the Gecko Nightly version instead.
 
 3. Make sure to select the correct build variant in Android Studio:
-**armDebug** for ARM
-**x86Debug** for X86
+**geckoBetaDebug** for the beta version of the Gecko engine
+**geckoNightlyDebug** for the nightly version of the Gecko engine
```

Comment:
I'd also add "recommended" on the geckoBetaDebug line.

Optional: could we also add a small section explaining what the other variants are? Or describe how this naming is made (e.g. gecko*Version*Fenix*Version*).